recipes/pcst-fast/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pcst-fast" %}
+{% set version = "1.0.10" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pcst_fast-{{ version }}.tar.gz
+  sha256: 3b5694110ce2e004471f383267d5e4ab7fe1ba9828954e8c42560ac1e42b25e6
+
+build:
+  number: 0
+  skip: True  # [py < 38]
+  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps --use-pep517 -vvv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - pip
+    - pybind11 >=2.1.0
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - pybind11 >=2.1.0
+    - python
+
+test:
+  imports:
+    - pcst_fast
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/fraenkel-lab/pcst_fast
+  license: MIT AND BSD-3-Clause AND Apache-2.0
+  license_family: MIT
+  license_file:
+    - LICENSE.txt
+    - external/pybind11/LICENSE
+    - external/googletest/googletest/LICENSE
+    - external/googletest/googlemock/LICENSE
+    - external/googletest/googlemock/scripts/generator/LICENSE
+  summary: "A fast implementation of the Goemans-Williamson scheme for the PCST (prize-collecting Steiner tree/forest) problem."
+
+extra:
+  recipe-maintainers:
+    - hag007
```

Comment:
Still need okay on this.

Reply:
Yep, I'll just remove them if I don't hear in the next few days. They have very little recent GH activity in recent months, so perhaps they are no longer active in maintaining recipes like this.